app/helpers/admin/site_maps_helper.rb

```diff
@@ -3,12 +3,26 @@
 module Admin::SiteMapsHelper
   include Admin::AdminHelper
 
+  def sitemap_pages
+    @sitemap_pages ||= Page.where(:_type.in => %w(Page SetPage)).entries
```

Comment:
Yeah ok, I can see the problem. So there's no way to tell if a page is a set entry just from its record in the db?